src/paginator.js

```diff
@@ -0,0 +1,300 @@
+import assign from 'lodash.assign';
+import omit from 'lodash.omit';
+
+
+export const styles = Object.freeze({
+  pageNumber: 1,
```

Comment:
It would maybe improve readability if these were both called `pageNumberPagination` and `limitOffsetPagination` or something along those lines to clearly indicate what they are.